variables.tf

```diff
@@ -27,7 +27,7 @@ variable "gsuite_admin_email" {
 
 variable "forseti_version" {
   description = "The version of Forseti to install"
-  default     = "v2.11.1"
+  default     = "v2.12.0"
```

Comment:
Since forseti-security hasn't tagged `v2.12.0` this has uncovered a bug - if the `forseti_version` can't be checked out it looks like we'll just check out the dev branch. Should the Forseti installation abort if an invalid version is provided?

Reply:
Good point- installation should abort if an invalid version is provided.